packages/webapp/public/locales/en/common.json

```diff
@@ -63,6 +63,7 @@
   "SUBMITTING": "Submitting...",
   "THATS_FINE": "That’s fine",
   "TODAY": "Today",
+  "TYPE": "Type",
```

Comment:
I didn't include the refactor to this common translation (it was too big to include) but it should be undertaken; there are 14 occurrences in `translation.json`